database/changesets/20201101_CommonTable.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog
+    xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+    xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext
+                        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd
+                        http://www.liquibase.org/xml/ns/dbchangelog
+                        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.8.xsd">
+    <changeSet author="jwarren@broadinstitute.org"
+               dbms="postgresql"
+               id="20201101"
+               logicalFilePath="20201101_CommonTable.xml">
+        <comment>
+            A table containing content common within workflows in a workload.
+        </comment>
+        <sql>
+            CREATE TABLE common_within_workload (
+                workload_id bigint NOT NULL,
+                workflow_options text NULL,
+
+                CONSTRAINT fk_workload_id
+                    FOREIGN KEY (workload_id)
+                    REFERENCES workload (id)
+                    ON DELETE CASCADE
+                    ON UPDATE CASCADE,
+                CONSTRAINT workload_id_unique
+                    UNIQUE (workload_id)
+            );
```

Comment:
Tests pass but does this look right @rexwangcc?

Reply:
It looks right to me. Let me test on your branch. I figured delete cascade is good since we are making a 1 to 1 mapping between records in `workload` table and `common` table, so it makes sense to delete the corresponding record from common when its workload is removed.

Reply:
Definitely, I literally didn't know that was a thing when I first wrote the liquibase stuff here

Reply:
OK, based on my local testing, both `ON UPDATE CASCADE` and `ON DELETE CASCADE` work.

Reply:
Thank you very very much for your lending your SQL prowess to making sure I'm not off my rocker--much more confident about this code now